db/migrate/20150316174702_create_reviews.rb

```diff
@@ -0,0 +1,15 @@
+class CreateReviews < ActiveRecord::Migration
+  def change
+    create_table :reviews do |t|
+      t.string :priority_id
+      t.string :status
```

Comment:
what's the status of a review? what's the difference between a status of a review and the status of its pickup?